temporal/api/cloud/usage/v1/message.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package temporal.api.cloud.usage.v1;
+
+option go_package = "go.temporal.io/api/cloud/usage/v1;usage";
+option java_package = "io.temporal.api.cloud.usage.v1";
+option java_multiple_files = true;
+option java_outer_classname = "MessageProto";
+option ruby_package = "Temporalio::Api::Cloud::Usage::V1";
+option csharp_namespace = "Temporalio.Api.Cloud.Usage.V1";
+
+import "google/protobuf/timestamp.proto";
+
+message Summary {
+    // Start of UTC day for now (inclusive)
+    google.protobuf.Timestamp start_time = 1;
+    // End of UTC day for now (exclusive)
+    google.protobuf.Timestamp end_time = 2;
+    // Records grouped by namespace
+    repeated RecordGroup record_groups = 3;
+    // True if data for given time window is not fully available yet (e.g. delays)
+    bool incomplete = 4;
+}
+
+message RecordGroup {
+    repeated GroupBy group_bys = 1;
```

Comment:
🤔 Curious about this usage. Is it intended that `GroupByKey` would become a logical `AND` if multiple are provided?

Reply:
Good question, that is correct. Currently it defaults to (and allows) only 1 key i.e. grouping by namespace. But multiple would mean logical `AND`